pageserver/src/deletion_queue.rs

```diff
@@ -160,11 +160,10 @@ pub struct DeletionQueueClient {
     lsn_table: Arc<std::sync::RwLock<VisibleLsnUpdates>>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
 struct TenantDeletionList {
     /// For each Timeline, a list of key fragments to append to the timeline remote path
     /// when reconstructing a full key
-    #[serde(serialize_with = "to_hex_map", deserialize_with = "from_hex_map")]
```

Comment:
Is this a breaking change? I would have expected TenantId TimelineId to already serialize as hex

Reply:
Nope, see https://github.com/neondatabase/neon/pull/5960 for context (there's a test to guarantee that this doesn't change serialization)